ssl/statem/extensions_srvr.c

```diff
@@ -1900,7 +1900,7 @@ EXT_RETURN tls_construct_stoc_early_data(SSL *s, WPACKET *pkt,
 
 #ifndef OPENSSL_NO_QUIC
         /* QUIC server must always send 0xFFFFFFFF, per draft-ietf-quic-tls-27 S4.5 */
-        if (s->quic_method != NULL)
+        if (SSL_IS_QUIC(s))
```

Comment:
This is already inside an `OPENSSL_NO_QUIC` block so I'm not sure it's strictly needed, but it's also harmless.

Reply:
Ah... that explains it ;)